src/walletClient/index.js

```diff
@@ -1347,6 +1347,56 @@ class ElvWalletClient {
     }
   }
 
+  async EntitlementMintingStatus({marketplaceParams, tenantId}) {
```

Comment:
shouldn't be necessary to have an entire separate method, the purpose of the MintingStatus method above is to call this status API and normalize the statuses - you can just add a case for 'nft-claim-entitlement' in there. Then EntitlementClaimStatus is just a filter on the op like all of the other user-facing status methods